spring-credhub-docs/build.gradle

```diff
@@ -63,17 +62,14 @@ asciidoctor {
 		}
 	}
 	logDocuments = true
-//	backends = ["html5"]
 	options doctype: 'book', eruby: 'erubis'
-	attributes  'icons': 'font',
+	attributes 'revnumber': project.version,
+			'spring-version': project.version,
```

Comment:
This doesn't seem right?

Reply:
It is what we have been using in other projects: https://github.com/spring-cloud/spring-cloud-app-broker/blob/main/spring-cloud-app-broker-docs/build.gradle#L93-L95 and what we have in `main`, but you are right, it doesn't seem correct to me, we should be using "${springVersion}" here.
We probably need to look better at docs publications and which numbers we are using, should we fix it as part of this or do we want to create a separate task to validate all the docs?

Reply:
My worry is that we are about to break it here, and what we have on `main` is wrong. @royclarkson do you have any context on this one?

Reply:
Agreed, the only benefit I see is that the project is not even building now anyway.
`spring-cloud-open-service-broker` is using the same property: https://github.com/spring-cloud/spring-cloud-open-service-broker/blob/main/spring-cloud-open-service-broker-docs/build.gradle#L90-L92

Reply:
That's a good point. Okay let's merge this to unblock and follow up on it.